extensions/amp-story-interactive/0.1/interactive-disclaimer.js

```diff
@@ -0,0 +1,168 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {closest} from '../../../src/dom';
+import {htmlFor} from '../../../src/static-template';
+import {setStyle} from '../../../src/style';
+
+/**
+ * Disclaimers will retrieve the information from the lookup dictionary below.
+ * In order to add a "Learn more" link or entity name ("Your response will be sent to <Organization>"),
+ * submit a PR with a new entry on the BACKENDS dictionary and tag @wg-stories to review it.
+ */
+
+/** @const {!Object<string, !Object>} */
+const BACKENDS = {
+  'webstoriesinteractivity-beta.web.app': {
+    learnMoreUrl: 'https://policies.google.com/terms',
+    entityName: 'Google Firebase',
+  },
+};
+
+/**
+ * Creates a disclaimer icon and dialog.
+ * @param {!Element} element
+ * @return {!Element}
+ */
+function buildDisclaimerLayout(element) {
+  const html = htmlFor(element);
+  return html`<div class="i-amphtml-story-interactive-disclaimer">
+    <div class="i-amphtml-story-interactive-disclaimer-content">
+      <div class="i-amphtml-story-interactive-disclaimer-icon"></div>
+      <div class="i-amphtml-story-interactive-disclaimer-bubble">
+        <div>
+          <span>Your response will be sent to </span>
+          <span class="i-amphtml-story-interactive-disclaimer-entity"></span>
+        </div>
+        <div class="i-amphtml-story-interactive-disclaimer-url"></div>
+        <div>
+          <a target="_blank" class="i-amphtml-story-interactive-disclaimer-link"
+            >Learn more</a
+          >
+        </div>
+        <button class="i-amphtml-story-interactive-disclaimer-close"></button>
+      </div>
+    </div>
+  </div>`;
+}
+
+/**
+ * Creates a disclaimer icon and dialog from the interactive element passed in.
+ * @param {!AmpStoryInteractive} interactive the interactive element.
+ * @return {!Element} the icon with the dialog that should be added to the shadowRoot.
+ */
+export function buildInteractiveDisclaimer(interactive) {
+  const {element} = interactive;
+  const backendUrl = element.getAttribute('endpoint').replace('https://', '');
+
+  const disclaimer = buildDisclaimerLayout(element);
+  const urlEl = disclaimer.querySelector(
+    '.i-amphtml-story-interactive-disclaimer-url'
+  );
+  const linkEl = disclaimer.querySelector(
+    '.i-amphtml-story-interactive-disclaimer-link'
+  );
+  const entityEl = disclaimer.querySelector(
+    '.i-amphtml-story-interactive-disclaimer-entity'
+  );
+
+  // Fill information
+  const backendSpecs = getBackendSpecs(backendUrl, BACKENDS);
+  interactive
+    .mutateElement(() => {
+      if (backendSpecs) {
+        entityEl.textContent = backendSpecs[1].entityName;
+        urlEl.textContent = backendSpecs[0];
+        backendSpecs[1].learnMoreUrl
+          ? (linkEl.href = backendSpecs[1].learnMoreUrl)
+          : linkEl.remove();
```

Comment:
If the link is not present it's effecting the scale in the closed state:
<img width="64" alt="Screen Shot 2020-12-11 at 11 40 50 AM" src="https://user-images.githubusercontent.com/3860311/101930173-ce32a400-3ba5-11eb-9bfb-9cbf4599680e.png">

Reply:
Left 2 `TODO`s on the CSS rules saying that the scale values depend on the size of the bubble. I have that working [here](https://github.com/ampproject/amphtml/pull/31534/files#diff-aaf53ed659819e653d5c15b8146e40e2e102886ad3375cc0cfcdb963b439bc0eR163) but it was also left out of this PR for simplicity.